tsconfig.json

```diff
@@ -10,16 +10,15 @@
     "allowSyntheticDefaultImports": true,
     "allowJs": false,
     "resolveJsonModule": true,
-    "noImplicitAny": true,
```

Comment:
Could you try without `any` ;) considers that such things have to be discussed separately I mean changing code style

Reply:
Yeah, but we've already had such option in config.